examples/stm32/xively_demo_stm32f2xx_nucleo_eth/Src/main.c

```diff
@@ -115,15 +115,14 @@ int main( void )
     /*##-1- Configure the UART peripheral ######################################*/
     /* Put the USART peripheral in the Asynchronous mode (UART Mode) */
     /* UART configured as follows:
-        - Word Length = 8 Bits (7 data bit + 1 parity bit) :
-                      BE CAREFUL : Program 7 data bits + 1 parity bit in PC HyperTerminal
+        - Word Length = 9 Bits (8 data bit + 1 parity bit) :
         - Stop Bit    = One Stop bit
         - Parity      = ODD parity
-        - BaudRate    = 9600 baud
+        - BaudRate    = 115200 baud
         - Hardware flow control disabled (RTS and CTS signals) */
     UartHandle.Instance        = USARTx;
-    UartHandle.Init.BaudRate   = 9600;
-    UartHandle.Init.WordLength = UART_WORDLENGTH_8B;
+    UartHandle.Init.BaudRate   = 115200;
+    UartHandle.Init.WordLength = UART_WORDLENGTH_9B;

```

Comment:
let's move this back to 8B and parity_none, but anyway the F2 project is not active we should consider to remove the whole project